website/content/stocks/fundamental_analysis/dcf/_index.md

```diff
@@ -1,12 +1,17 @@
 ```text
-usage: dcf [-a] [-h]
+usage: dcf [-a] [--no-ratios] [-p NUM] [-s NUM] [-h]
 ```
 
 Generates a completed discounted cash flow statement as an excel spreadsheet export. The statement uses machine learning to predict future financial statements and share price based on the predicted financials. Source: https://stockanalysis.com/stocks/
 
 ```
 optional arguments:
   -a, --audit  Confirms that the numbers provided are accurate. (default: False)
```

Comment:
Could you elaborate on this? This seems like a no brainer that this should be **true** by default.

Reply:
So the audit is almost useless for most users. All it does is add cells below the the statement of cash flows that make sure the columns that need to be summed equal the sum column, by showing the difference. Great for development or for people who don't trust StockAnalysis, but no real value add most of the time.

Reply:
I'd suggest renaming it in that case or adding info that it is just for development. It now reads like a no-brainer that it should be enabled.

Reply:
Definitely does sound that way, I updated with with some better wording.